linter.py

```diff
@@ -12,5 +16,57 @@ class Phpmd(Linter):
     tempfile_suffix = 'php'
     defaults = {
         'selector': 'source.php, text.html.basic',
-        '@rulesets:,': 'cleancode,codesize,controversial,design,naming,unusedcode'
     }
+
+    default_ruleset = 'cleancode,codesize,controversial,design,naming,unusedcode'
+
+    def cmd(self):
+
+        settings = Linter.get_view_settings(self)
```

Comment:
Always use `self.method()` not `class.method(self)`. Also no empty lines after a `def`.

Reply:
Fixed.